packages/base/Tagselector/src/TagSelector/TagSelector.tsx

```diff
@@ -71,10 +71,12 @@ export interface Props
   otherOptionLabel?: string
   /** Callback invoked when other option selected */
   onOtherOptionSelect?: (value: string) => void
+  /** Select other option on Enter key press */
+  submitOtherOptionOnEnter?: boolean
   /** Allow to show the other option in the list of options */
   showOtherOption?: boolean
   /** Label to show when no options were found */
-  noOptionsText?: string
+  noOptionsText?: string | null
```

Comment:
Why do we need both `?:` and `null` here? Wouldn't the property being optional already suffice?

Reply:
Using `null` as a semantic symbol is a bit confusing, specially when paired with `undefined`. I usually expect both `undefined` and `null` to have the same meaning. Maybe we could use `false` here as an alternative? Wdyt? It is usually what is used on other component APIs to signify that a prop should be disabled
```suggestion
  noOptionsText?: string | false
```

Reply:
@augustobmoura I think in this case this could be addressed in a separate ticket, as this is just a refactoring that tries not to break existing API, what do you think?

Reply:
@augustobmoura I would prefer to keep it consistent with `options` (that can also be `null`), otherwise we will have a breaking change, what do you think? I agree that `null` is a bit cryptic, but this seems to be a change out of scope

<img width="403" alt="Screenshot 2024-09-10 at 13 00 10" src="https://github.com/user-attachments/assets/e743dba6-83d2-4dc4-b1ef-5e5078673850">

Reply:
I don't know, using `null` is not really a pattern in Picasso, except for a few exceptions (this being one of them), most of the code uses optional properties at best.

Can we update the documentation for these props, then? Explaining that `null` is a special case and passing it to the props will effectively disable them

Reply:
@augustobmoura Documentation was updated, thanks! But for `options: null` there is nothing to add, as it is the same as `options: []`

Reply:
>But for options: null there is nothing to add, as it is the same as options: []

That is an example of an unnecessary `null` sigil
We can probably use `false` now, since the argument for consistency doesn't hold up anymore (`options` is not using `null` for the same reason as `noOptionsText`)

I'm already satisfied with the documentation for `null`, so no need to refactor it to `false`. But it is something to keep in mind in the future

Reply:
@augustobmoura Agree!